x-pack/plugins/observability_solution/entity_manager/server/lib/entities/install_entity_definition.ts

```diff
@@ -218,3 +188,123 @@ async function installAndStartDefinition(
   }
   return definition;
 }
+
+// perform installation of an entity definition including all
+// the necessary components
+async function install({
```

Comment:
Does this function need to be wrapped to catch the error thrown to update the status as `failed`?